docs/config-file.md

```diff
@@ -110,3 +125,18 @@ Every protocol has different configuration needs so each protocol has unique con
 |mqtt.authentication.userPassword| username and password parameters for authentication|
 |mqtt.authentication.userPassword.username| username parameter
 |mqtt.authentication.userPassword.password| password parameter
+
+### Kafka
+
+|Field|Description|
+|---|---|
+|kafka.authentication| Kafka authentication configuration|
+|kafka.authentication.key | Kafka Broker Key
+|kafka.authentication.cert| Client certificate
+|kafka.authentication.clientId| Kafka client Id for authentication
+|kafka.authentication.rejectUnauthorized | Boolean flag for accepting the valid SSL certificates
+|kafka.authentication.userPassword| username and password parameters for authentication|
+|kafka.authentication.userPassword.username| username parameter
+|kafka.authentication.userPassword.password| password parameter
+|kafka.authentication.scramSha256.mechanism | ScramSha256 SASL mechanism type for authentication
+|kafka.authentication.scramSha512.mechanism | ScramSha512 SASL mechanism type for authentication
```

Comment:
```suggestion
|kafka.authentication.username| The username to use during authentication.
|kafka.authentication.password| The password to use during authentication.
```

Let's simplify username and password and remove the scram mechanism as it's coming from the AsyncAPI file, not from the config file.